sdk/keyvault/azcertificates/test-resources.json

```diff
@@ -0,0 +1,331 @@
+            ],
+            "properties": {
+                "publicAccess": "None"
+            }
+        },
+        {
+
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2020-12-01",
+            "name": "[variables('attestationFarm')]",
+            "condition": "[parameters('enableHsm')]",
+            "location": "[parameters('location')]",
+            "kind": "linux",
+            "sku": {
+                "name": "B1"
+            },
+            "properties": {
+                "reserved": true
+            }
+        },
+        {
+
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2020-12-01",
+            "name": "[variables('attestationSite')]",
+            "condition": "[parameters('enableHsm')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', variables('attestationFarm'))]"
+            ],
+            "location": "[parameters('location')]",
+            "properties": {
+                "httpsOnly": true,
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('attestationFarm'))]",
+                "siteConfig": {
+                    "name": "[variables('attestationSite')]",
+                    "alwaysOn": true,
+                    "linuxFxVersion": "[variables('attestationUri')]",
+                    "appSettings": [
+                        {
+                            "name": "WEBSITES_ENABLE_APP_SERVICE_STORAGE",
+                            "value": "false"
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "outputs": {
+        "AZURE_KEYVAULT_URL": {
+            "type": "string",
+            "value": "[reference(variables('kvName')).vaultUri]"
+        },
+        "AZURE_MANAGEDHSM_URL": {
+            "type": "string",
+            "condition": "[parameters('enableHsm')]",
+            "value": "[reference(variables('hsmName')).hsmUri]"
+        },
+        "KEYVAULT_SKU": {
+            "type": "string",
+            "value": "[reference(parameters('baseName')).sku.name]"
+        },
+        "CLIENT_OBJECTID": {
+            "type": "string",
+            "value": "[parameters('testApplicationOid')]"
+        },
+        "BLOB_STORAGE_ACCOUNT_NAME": {
+            "type": "string",
+            "value": "[variables('primaryAccountName')]"
+        },
+        "BLOB_PRIMARY_STORAGE_ACCOUNT_KEY": {
+            "type": "string",
+            "value": "[listKeys(variables('primaryAccountName'), variables('mgmtApiVersion')).keys[0].value]"
+        },
+        "BLOB_CONTAINER_NAME" : {
+            "type": "string",
+            "value": "[variables('blobContainerName')]"
+        },
+        "AZURE_KEYVAULT_ATTESTATION_URL": {
+            "type": "string",
+            "condition": "[parameters('enableHsm')]",
+            "value": "[format('https://{0}/', reference(variables('attestationSite')).defaultHostName)]"
+        }
+    }
+}
```

Comment:
Can you convert this to bicep? We should be using bicep going forwards and I'd love for the go repo to be a good example of that. I can help with conversion/generation if you need it. `az bicep decompile -f test-resources.json`

Reply:
This is a copy of what all other languages use. Normally I create/update this for KV and everyone just copies it. So it's not really a new definition. I'd like to do this for all other services, too. It's not an efficient use of time for every language to maintain their own test-resources.json (and related files).

I plan to convert this to bicep at some point, but it's not been a priority. The ARM templates work just fine and so far we've only needed simple incremental changes on top them.

Reply:
Ok let's just treat the keyvault ones as a batch then. I'll approach the conversion on a service to service basis instead of a language to language basis.

Reply:
I think it's also completely valid to encourage conversion. That said, I'm typically very much of the mind: if it ain't broke, don't fix it. But for new services we should perhaps encourage it through PRs.